src/code/z_actor.c

```diff
@@ -3631,7 +3644,7 @@ void func_8003424C(PlayState* play, Vec3f* arg1) {
 }
 
 void Actor_SetColorFilter(Actor* actor, s16 colorFlag, s16 colorIntensityMax, s16 xluFlag, s16 duration) {
```

Comment:
xluFlag could be renamed to bufFlag maybe (as in, opa buffer, xlu buffer)
mostly suggesting it because `xluflag_opa` looks weird